terraform/environments/core-shared-services/instance-scheduler-lambda-function.tf

```diff
@@ -3,7 +3,7 @@ module "instance_scheduler" {
   #checkov:skip=CKV_AWS_117
   #checkov:skip=CKV_AWS_272 "Code signing not required"
   #checkov:skip=CKV_AWS_173 "These lambda envvars aren't sensitive and don't need a cmk. Default AWS KMS key is sufficient"
-  source                         = "github.com/ministryofjustice/modernisation-platform-terraform-lambda-function?ref=20e0d9e27402d1e012159a41b474da908d74941b" #v2.0.0
+  source                         = "github.com/ministryofjustice/modernisation-platform-terraform-lambda-function?ref=5a3c02a071519986a0ae415168fb4f9d3fb7970f" #v2.0.0
```

Comment:
just double checking the version shouldn't be... "github.com/ministryofjustice/modernisation-platform-terraform-lambda-function?ref=a4392c1cc00014a274680d160e75464dce4ba919" #v2.1

Reply:
there is no release of the lambda yet (I want to test it first and update unit tests), hence it's the latest commit, but no new tag